packages/turf-voronoi/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@turf/voronoi",
+  "version": "0.1.0",
+  "description": "turf module to generate Voronoi polygons from a set of points and a bounding box",
+  "main": "main",
+  "module": "index",
+  "jsnext:main": "index",
+  "files": [
+    "index.js"
```

Comment:
You'll also need to provide a Typescript definition, `index.d.ts` and the `main.js` (Rollup build output).
Look at any other TurfJS `package.json` for examples.

Reply:
Ok, added - but I have no idea what I'm doing here, or how this file is used. This would be a good thing to document in CONTRIBUTING.md.